frontend/src/metabase/query_builder/selectors.unit.spec.js

```diff
@@ -133,6 +142,41 @@ describe("getIsResultDirty", () => {
       expect(getIsResultDirty(state)).toBe(true);
     });
 
+    it("converts clauses into plain MBQL objects", () => {
+      const aggregation = ["count"];
+      const breakout = ["field", ORDERS.CREATED_AT.id, null];
+      const filter = [">=", ["field", ORDERS.TOTAL.id, null], 20];
```

Comment:
fwiw, `metabase-lib/metadata/Field` instances should have a `reference` method that could give you a simple field ref. It should be possible to replace this with something like that:

```jsx
const breakout = ORDERS.CREATED_AT.reference()
const filter = [">=", ORDERS.TOTAL.reference(), 20]
```